### PR TITLE
Updated FAQ content

### LINKED ISSUE #35: Review FAQ page to make it easier to read

### PR DESCRIPTION
**Pull request by zarino**
Closes #35.

image

### CODE REVIEW

File: templates/web/smidsy/about/faq.html [hunk 1]
```diff
@@ -3,95 +3,174 @@
...
+
+    <dt>Does all the data come from reports made on this site?</dt>
+    <dd>No: By default, Collideoscope <em>also</em> displays all incidents involving bicycles from the Department for Transport’s yearly STATS19 dataset. You can choose to exclude STATS19 reports, though, if you’d prefer to see only reports made by Collideoscope users.</dd>
+
+    <dt>What is STATS19?</dt>
+    <dd>STATS19 is the name of the form on which police officers in Britain record incidents on the public highway involving human injury or death. This data is published annually by the Department for Transport. We aim to keep Collideoscope up to date with the latest available STATS19 data – right now that means cycling accidents that occured in 2012–2016. Accidents from 2017 should become available in late 2018.</dd>
+</dl>
+
+<h2 name=feedback">Feedback and ideas</h2>
```
Comment (dracos) [inline templates/web/smidsy/about/faq.html]: Typo in attribute here.

File: templates/web/smidsy/about/faq.html [hunk 2]
```diff
@@ -3,95 +3,174 @@
...
-        <dd>
-            <p>This website is our attempt to enable society to collaborate in order to collect some much-needed evidence to inform the ongoing cycle-safety debate. Our aim is to build a map of cycling-related incidents and near-misses for the UK so that transport planners, health researchers and decision-makers can see for the first time where incidents are happening, how often they take place, and how serious they are. We believe this kind of evidence fills a gap that will potentially enable local Councils to make proactive changes to highway environments so as to prevent serious, and potentially fatal, cycling incidents from occurring in the future.</p>
-        </dd>
-    </dl>
+
+<h2 name="general">General</h2>
+
+<dl>
+    <dt>What is this site for?</dt>
+    <dd>Collideoscope invites you to report cycling collisions and near misses in the UK. It collects the reports together and makes the data available to planners, researchers and campaigners with the aim of making our roads safer for all.</dd>
```
Comment (dracos) [inline templates/web/smidsy/about/faq.html]: Could we wrap this to make it easier to read in e.g. github PR reviews? :)

File: templates/web/smidsy/about/faq.html [hunk 3]
```diff
@@ -3,95 +3,174 @@
...
-        <dd>Collideoscope was built by <a href="http://www.mysociety.org/">mySociety</a> and <a href="http://itpworld.net">Integrated Transport Planning Ltd</a>. It is based upon the FixMyStreet project, <a href="http://www.fixmystreet.com/faq#organisation">See full credits for FixMyStreet</a>.
-        </dd>
-
-        <dt>Why build Collideoscope?</dt>
-        <dd>
-            <p>This website is our attempt to enable society to collaborate in order to collect some much-needed evidence to inform the ongoing cycle-safety debate. Our aim is to build a map of cycling-related incidents and near-misses for the UK so that transport planners, health researchers and decision-makers can see for the first time where incidents are happening, how often they take place, and how serious they are. We believe this kind of evidence fills a gap that will potentially enable local Councils to make proactive changes to highway environments so as to prevent serious, and potentially fatal, cycling incidents from occurring in the future.</p>
-        </dd>
-    </dl>
+
+<h2 name="general">General</h2>
```
Comment (dracos) [inline templates/web/smidsy/about/faq.html]: Think you mean `id`.

File: templates/web/smidsy/about/faq.html [hunk 4]
```diff
@@ -3,95 +3,174 @@
...
+
+<h2 name="reporting">Making a report</h2>
+
+<dl>
+    <dt>What kind of reports can be made on Collideoscope?</dt>
+    <dd>You can report actual collisions between a bicycle and another vehicle, or near misses where an incident was averted. Additionally you may report incidents where no other vehicle was involved, such as collisions or near misses with pedestrians, street furniture... or anything else.</dd>
+    <dd>Please <strong>do not report incidents requiring immediate medical or police attention</strong> — or at least, not until you have requested help via the emergency services’ official channels.</dd>
+
+    <dt>Where do reports go?</dt>
+    <dd>Reports are sent to the local authority with responsibility for the area in which the incident occured. Additionally, the data is published on this site and is available to anyone who might like to use it, such as researchers, town planners, campaigners and local residents. <a href="/reports">Find out more.</a></dd>
```
Comment (dracos) [inline templates/web/smidsy/about/faq.html]: I don't like full stops being part of the link, is that just me?

File: templates/web/smidsy/about/faq.html [hunk 5]
```diff
@@ -3,95 +3,174 @@
...
+    <dd>Try not to include personally-identifiable information (such as people’s names or addresses, or vehicle license plate numbers) in your report.</dd>
+
+    <dt>Why can’t my report include a photo or video?</dt>
+    <dd>We don’t provide this facility on Collideoscope, because if you are intending to also send a report to the police, putting video evidence of your incident online can compromise any subsequent court hearing. Additionally, where a vehicle’s numberplate is clearly shown, associating the photograph to public accusations of wrongdoing may constitute libel. Police say that they often find this type of evidence invaluable, but it should be relayed to them directly and not shared in a public arena where others can comment on it.</dd>
+
+    <dt>What happens next?</dt>
+    <dd>Once you have filed your report via Collideoscope, it will immediately appear on the website. At the same time, the details will be sent off to your local council, who may or may not wish to follow up with you. After a month, we’ll send an automated follow-up to ask whether anything happened in response to your report.</dd>
```
Comment (dracos) [inline templates/web/smidsy/about/faq.html]: Follow up only a maybe - if police should be involved.

File: templates/web/smidsy/about/faq.html [hunk 6]
```diff
@@ -3,95 +3,174 @@
...
+
+<h2 name="issues">When things don’t go as planned</h2>
+
+<dl>
+    <dt>Collideoscope doesn’t recognise the postcode I’m trying to input</dt>
+    <dd>Our postcode data comes from Ordnance Survey. If you have a relatively new postcode, it may not yet be reflected on OS, and therefore on Collideoscope.</dd>
+    <dd>It’s ok though, you can still use the site: just enter a nearby place name, street name or let the site locate you automatically, instead. You can use the pan and zoom controls at the top right hand corner of the screen to zero in on the precise location.</dd>
+
+    <dt>My street isn’t on your map</dt>
+    <dd>Collideoscope’s default maps are from Ordnance Survey, so updates to our maps happen when OS refresh their own data. If you can’t see the street you want to make a report on, your best bet is to use the map to pinpoint the location as accurately as you can, then describe the problem location in as much detail as possible in the description box.</dd>
```
Comment (dracos) [inline templates/web/smidsy/about/faq.html]: No, they're OpenStreetMap.

File: templates/web/smidsy/about/faq.html [hunk 7]
```diff
@@ -3,95 +3,174 @@
...
+
+<h2 name="accounts">Accounts</h2>
+
+<dl>
+    <dt>Do I need to register with Collideoscope to use it?</dt>
+    <dd>No – you can send a report without registering. When you submit your report, we’ll send you an email with a confirmation link.</dd>
+    <dd>If you use Collideoscope regularly, you can avoid this step by creating an account. Once you’ve signed in, your reports will be sent as soon as you click ‘submit’.</dd>
+    <dd>You’ll also be able to see all previous reports you’ve made on one page.</dd>
+
+    <dt>How do I register?</dt>
```
Comment (dracos) [inline templates/web/smidsy/about/faq.html]: I would ditch this question (at least, until we update the auth page :) ), it is confusing.

File: templates/web/smidsy/about/faq.html [hunk 8]
```diff
@@ -3,95 +3,174 @@
...
+
+<dl>
+    <dt>I work for a council. How can we access data from Collideoscope?</dt>
+    <dd>If your council is within Great Britain, we are already sending reports to you as soon as they are submitted by our users. Please <a href="/contact">drop us a line</a> if you’d like to check or change which email address we are using, or if you’d like an address added.</dd>
+
+    <dt>Can we get more or different data in reports?</dt>
+    <dd><a href="/contact">Contact us</a> to let us know what you need; it’s possible we’ll be able to add to or modify the report form for your area.</dd>
+
+    <dt>Can we modify the text you send to people who make reports?</dt>
+    <dd>Yes! If there’s something particular you’d like to include about, for example, your commitment to road safety, or plans for future improvements, just drop us a line and we can change our emails accordingly.</dd>
```
Comment (dracos) [inline templates/web/smidsy/about/faq.html]: This facility doesn't currently exist (I don't know if we want to leave this text here to see if anyone asks for it).

File: templates/web/smidsy/about/faq.html [hunk 9]
```diff
@@ -3,95 +3,174 @@
...
+</dl>
+
+<h2 name="data">Collideoscope data</h2>
+
+<dl>
+    <dt>What kind of data can I find on this site?</dt>
+    <dd>Take a look for yourself, on the <a href="/reports">Statistics page</a>. You can filter the charts and graphs on the page by multiple criteria, including location, date, and the bodies involved in the incident. You can also enter a council name to see a map of reports in that area. And finally, you can download an anonymised copy of the data as a CSV, perfect for analysing in Excel or Fusion Tables.</dd>
+
+    <dt>Does all the data come from reports made on this site?</dt>
+    <dd>No: By default, Collideoscope <em>also</em> displays all incidents involving bicycles from the Department for Transport’s yearly STATS19 dataset. You can choose to exclude STATS19 reports, though, if you’d prefer to see only reports made by Collideoscope users.</dd>
```
Comment (dracos) [inline templates/web/smidsy/about/faq.html]: The default (on the map) does not appear to include stats19 reports.

File: templates/web/smidsy/about/faq.html [hunk 10]
```diff
@@ -3,95 +3,174 @@
...
+
+    <dt>What is STATS19?</dt>
+    <dd>STATS19 is the name of the form on which police officers in Britain record incidents on the public highway involving human injury or death. This data is published annually by the Department for Transport. We aim to keep Collideoscope up to date with the latest available STATS19 data – right now that means cycling accidents that occured in 2012–2016. Accidents from 2017 should become available in late 2018.</dd>
+</dl>
+
+<h2 name=feedback">Feedback and ideas</h2>
+
+<dl>
+    <dt>I’d like to make a Collideoscope for my own country</dt>
+    <dd>Collideoscope runs on open-source code, so that’s fine. It is a repurposing of the FixMyStreet codebase: <a href="https://fixmystreet.org/">read more here</a> and then <a href="/contact">get in touch</a> if you’d like some help.</dd>
```
Comment (dracos) [inline templates/web/smidsy/about/faq.html]: I would drop get in touch if you need help, I assume!

File: templates/web/smidsy/about/faq.html (hunk 4 above)
Comment (zarino) [reply templates/web/smidsy/about/faq.html]: IIRC @MyfanwyNixon’s preference is also for "whole sentence" links to include the full stop. So I think we’re keeping this.

File: templates/web/smidsy/about/faq.html (hunk 8 above)
Comment (zarino) [reply templates/web/smidsy/about/faq.html]: Yeah, I figured we’d leave this, just to see whether someone wants it. Given sufficient time/budget, we could implement it!

File: templates/web/smidsy/about/faq.html (hunk 3 above)
Comment (zarino) [reply templates/web/smidsy/about/faq.html]: Ah yes, sorry, I dropped the `<a>` tags, but forgot to swap to `id`s instead. Thanks for spotting! :+1:

**Pull request merged**